templates/dag/dagman_wrapper.sh

```diff
@@ -5,6 +5,5 @@
 
 
 # touch our transfer files so condor will copy them back
-(sleep 1; touch {%for f in transfer_files%}{{f}} {%endfor%}) &
```

Comment:
Sorry if it's my ignorance here, but what does the dash at the beginning of the command do?

Reply:
That's just the minus in the diff. for the line being deleted.

We don't need this touch stuff anymore, as we are transferring all those files to dcache at submission time;  the touch was so the files would get transferred back by condor_transfer_data...

Reply:
OMG.  I feel so dumb.  Time to look away from the screen for a bit.